components/Dialogs/README.md

```diff
@@ -187,6 +188,49 @@ MDCAlertAction *alertAction =
 
 ## Extensions
 
+<!-- Extracted from docs/theming.md -->
+
+### Theming
+
+You can theme an MDCDialog to match the Material Design Dialog using your app's schemes in the DialogThemer
+extension.
+
+You must first add the DialogThemer extension to your project:
+
+```bash
+pod 'MaterialComponents/Dialogs+DialogThemer'
+```
+
+You can then import the extension and create an `MDCAlertControllerThemer` instance. A dialog scheme defines
+the design parameters that you can use to theme your dialogs.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+// Step 1: Import the DialogThemer extension
+import MaterialComponents.MaterialDialogs_DialogThemer
+
+// Step 2: Create or get a alert scheme
```

Comment:
tip: "an alert" (missing "n").
(and same for the obj-c step 2 below).

Reply:
Any changes should be made upstream (in the develop branch).